runtime/src/main/java/org/corfudb/runtime/view/SequencerView.java

```diff
@@ -92,6 +92,18 @@ public TokenResponse next(UUID ... streamIds) {
         }
     }
 
+    /**
+     * Return the next ordered globally unique sequence number from the sequencer.
+     *
+     * @return A token that can serve as the globally ordered unique sequence number.
+     */
+    public TokenResponse nextOrderedGuid() {
```

Comment:
Try to avoid using CFUtils, you should use CompletableFutures instead. I can help to figure out how to change this piece of code

Reply:
Its copy pasted code :( I may need to refactor entire RPC layer